src/broadcast/api.ts

```diff
@@ -58,8 +59,11 @@ const broadcastApi: BroadcastService = {
     });
     return destroy;
   },
-  async refreshBroadcastList(authToken: string): Promise<void> {
-    await ipc_refreshBroadcastList.renderer!.trigger({ authToken });
+  async connect(authToken: string): Promise<void> {
```

Comment:
```suggestion
  async connectToSpectateServer(authToken: string): Promise<void> {
```
Think this method needs to be updated too

Reply:
done